Automattic/jetpack
# Protect: Fix Modal overflow scrolling

Fixes #
When the Modal component's height overflows the viewport, there is no way to scroll to access elements that are out of view. 

## Proposed changes:

* Add `overflow-y: auto` to the `Modal` component's primary `.modal` class to allow the contents to remain accessible by scrolling when they extend out of view.

### Other information:

## Jetpack product discussion


- Automattic/jetpack-scan-team#829

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Create a Jurassic Ninja site with Protect and the Beta Tester plugins installed and a Scan plan
* Add a variety of fixable threats and allow the initial scan to process
* From the Protect UI, select Fix all threats
* When prompted for your credentials reduce your viewport height drastically so that the screen in smaller than the height of the Modal
* Attempt to scroll the Modal to access the button in the lower portion
* Activate this branch from the Beta Tester plugin
* Proceed back to the Protect and once again attempt to scroll the Modal to the lower portion with a limited viewport height
* Validate your connection and credentials and return to the Protect UI
* Ensure that even with a significant number of fixable threats you can continue to access the buttons at the lower part of the Modal despite a limited viewport
* Ensure that no regressions are introduced

## Pictures

Before:
![Screen Capture on 2023-12-05 at 14-52-06](https://github.com/Automattic/jetpack/assets/43220201/b719a570-924a-4ff7-a5f8-376a42e10718)

After:
![Screen Capture on 2023-12-05 at 14-53-55](https://github.com/Automattic/jetpack/assets/43220201/d39fc417-974f-4564-9995-97338adade00)
